### DISCUSSION TITLE
log unsuccessful login atttempts

### DISCUSSION THREAD

**Discussion by yurividal** [linuxserver/Heimdall#983] [Ideas]
is it possible to log unsuccessful  login atttempts? my goal is to use fail2ban to ban any ip addresses that try to login to my heimdal and fails more than 3 times, but heimdal doesn't seem to produce any logs of logins

**Comment by keriati** [comment]
Right now Heimdall already throttles the login attempts using Laravel's built in ThrottlesLogins trait.
